### PR TITLE
Issue296

### PR DESCRIPTION
**Pull request by DylanABacon**
originates from SMC-652

### CODE REVIEW

**Comment by JensEggers** [comment]
why is this work being done in a fork?

File: lib/modules/rest/index.js [hunk 1]
```diff
@@ -325,8 +325,16 @@ Rest.prototype.handleRequest = function(req, res, $happn, $origin) {
               return;
             }
 
-            if (body.parameters[parameter.name]) args.push(body.parameters[parameter.name]);
-            else args.push(null);
+            if (body.parameters[parameter.name]) {
```
Comment (JensEggers) [inline lib/modules/rest/index.js:328]: to be safe, check for body.parameters first:
```js
if (body.parameters && body.parameters[parameter.name]) {
```

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "happner-2",
-  "version": "11.3.0",
+  "version": "11.3.1",
```
Comment (JensEggers) [inline package.json:3]: this is a new feature, rather go to 11.4.0

File: lib/modules/rest/index.js (hunk 1 above)
Comment (DylanABacon) [reply lib/modules/rest/index.js:328]: Just before this map, in ln303, body.parameters is defined as an empty object if it does not exist.

File: package.json (hunk 2 above)
Comment (DylanABacon) [reply package.json]: updated, thanks

**Comment by DylanABacon** [comment]
> why is this work being done in a fork?

I dont have access to the origin repo. If this PR is approved, Ill request a merge of this fork into the origin repo

**Pull request merged**